python/CHANGES.rst

```diff
@@ -0,0 +1,15 @@
+snowballstemmer 1.9.1, 2019-09-05
+=================================
+
+* Added Hindi stemmer (#73).
+* Added Basque and Catalan stemmers, thanks Israel Olalla (#104).
+* Improved Greek stemmer.
+* Fixed memory leak when processing ``among`` commands.
```

Comment:
That's only in the snowball compiler, not the generated code for Python (or any other language).  I wouldn't term it a memory "leak" either, since the compiler exits right after this memory is released at which point the OS will reclaim all the memory from the process - this change really just gets rid of a message about "unfreed blocks".

Reply:
OK, I will remove this line.

Reply:
Too late as you merged this. Maybe you can remove it yourself so that I don't need a new pull request?

Reply:
OK, removed in b41793512d3880294af75f157f4b2154fc011060.